### PR TITLE
chore(26919): bump `@metamask/signature-controller` to 19.0.0 and `@metamask/logging-controller` to 6.0.0

### LINKED ISSUE #26919: LoggingController and SignatureController upgrade
### What is this about?

The `LoggingController` is behind by one major version behind and needs to be updated.
The `SignatureController` is behind by one major version behind and needs to be updated.

These two controllers are peer dependencies so they make sense to be grouped together. 

### Scenario

_No response_

### Design

_No response_

### Technical Details

_No response_

### Threat Modeling Framework

_No response_

### Acceptance Criteria

- `LoggingController` is on v6
- `SignatureController` is on v19

### Stakeholder review needed before the work gets merged

- [X] Engineering (needed in most cases)
- [ ] Design
- [ ] Product
- [ ] QA (automation tests are required to pass before merging PRs but not all changes are covered by automation tests - please review if QA is needed beyond automation tests)
- [ ] Security
- [ ] Legal
- [ ] Marketing
- [ ] Management (please specify)
- [ ] Other (please specify)

### References

_No response_

### PR DESCRIPTION
**Pull request by DDDDDanica**
## **Description**
In order to achieve core release v194, this PR bumps `@metamask/signature-controller` to 19.0.0 and `@metamask/logging-controller` to 6.0.0.
ChangeLog for each controllers are:
- https://github.com/MetaMask/core/blob/main/packages/logging-controller/CHANGELOG.md
- https://github.com/MetaMask/core/blob/main/packages/signature-controller/CHANGELOG.md

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/27082?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/26919

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

**Comment by DDDDDanica** [comment]
@metamaskbot update-policies

File: app/scripts/metamask-controller.js [hunk 1]
```diff
@@ -399,6 +399,7 @@ export default class MetamaskController extends EventEmitter {
     this.loggingController = new LoggingController({
       messenger: this.controllerMessenger.getRestricted({
         name: 'LoggingController',
+        allowedActions: ['LoggingController:getState'],
```
Comment (MajorLift) [inline app/scripts/metamask-controller.js:402]: Hi @DDDDDanica, thanks for your work on this! I had two notes on conventions for instantiating messengers:

- Internal actions/events (i.e. in this case all message types prefixed with the `LoggingController:` namespace, including `LoggingController:getState`) should be excluded from a restricted messenger's allowlists. The idea is that internal message types are always supported by a messenger, so they don't have to be specified. 

- `LoggingControllerMessenger` is defined with empty allowlists for both external actions and events. For empty allowlists, we assign an empty array instead of omitting the property (if you're interested, here's some discussion on why we enforce this: https://github.com/MetaMask/core/issues/4033).

So in conclusion, all we need to do here is the following:

```suggestion
        allowedActions: [],
        allowedEvents: [],
```

Comment (DDDDDanica) [reply app/scripts/metamask-controller.js]: Hey @MajorLift thanks so much for reviewing and both notes make perfect sense !  I've pushed 2 empty arrays for `LoggingController` - [3443cc](https://github.com/MetaMask/metamask-extension/pull/27082/commits/3443cc7e22292809a3eb8dca40bbdc180938bc5b)

Comment (MajorLift) [reply app/scripts/metamask-controller.js]: Looks good!

**Review by vthomas13 — APPROVED** [review]
Looks great! Thanks for going over this with me @DDDDDanica

**Review by MajorLift — APPROVED** [review]
LGTM!
